web/react/utils/utils.jsx

```diff
@@ -527,6 +527,25 @@ export function splitFileLocation(fileLocation) {
     return {ext: ext, name: filename, path: filePath};
 }
 
+export function getPreviewImagePath(filename) {
+    // Returns the path to a preview image that can be used to represent a file.
+    const fileInfo = splitFileLocation(filename);
+    const fileType = getFileType(fileInfo.ext);
+
+    if (fileType === 'image') {
+        // This is a temporary patch to fix issue with old files using absolute paths
```

Comment:
I feel like I've moved this "temporary" fix around 3 or 4 times so I'm good with that